Cue/app/tabs/library/play/PlayDeckView.js

```diff
@@ -145,6 +153,24 @@ class PlayDeckView extends React.Component {
     StatusBar.setHidden(false)
   }
 
+  _flaggedOnly(array: Array<*>): Array<*> {
+    return array.slice(0).filter(item => item.needs_review)
+  }
+
+  _answerFirst(array: Array<*>): Array<*> {
+    let ret = []
+
+    for (let i = 0; i < array.length; i++) {
+      ret.push({
+        ...array[i],
+        front: array[i].back,
+        back: array[i].front
+      })
+    }
+
+    return ret
```

Comment:
You should be able to simplify this to `return array.map(deck => {...deck, front: deck.back, back: deck.front})`?